### ISSUE TITLE
Broken compaction tests

### ISSUE THREAD

**Issue by DomGarguilo** [apache/accumulo#4135]
**Test name(s)**
- ExternalCompaction4_IT.testErrorDuringCompactionNoOutput
- CompactionIT.testErrorDuringCompactionNoOutput
- CompactionIT.testMultiStepCompactionThatDeletesAll

**Describe the failure observed**
It looks like the mentioned tests are failing after #4127 was merged into 2.1. I'm not sure if those changes caused a bug or if it changed behavior that these tests were expecting.

**Testing Environment:**
 - Version of this project: 2.1
 - First commit known to fail (or current commit): 08628c016bdc8597de973cf3f4fe04907a43fc7d

**What have you tried already?**
Tried running the failing tests before this commit and they pass.

**Comment by DomGarguilo** [comment]
Fixed in #4134

**Comment by ctubbsii** [comment]
Haven't seen this in awhile. Can reopen if it's still an issue.
